src/components/ProductSummary.vue

```diff
@@ -1,32 +1,76 @@
 <template>
-    <v-sheet class="d-flex" :elevation="1" border>
-        <div class="d-inline-flex ma-2">
-            <v-img
-                :width="240"
-                aspect-ratio="4/3"
-                src="https://cdn.vuetifyjs.com/images/parallax/material.jpg"
-            ></v-img>
-        </div>
-        <div class="d-flex flex-column ma-2">
-            <router-link :to="link">
-                <span class="text-body-1">{{ internalName }}</span>
-            </router-link>
-
-            <ProductPrice :price="price" :retail-price="retailPrice" />
+    <v-sheet :border="true" :elevation="1">
+        <div class="d-flex justify-space-between ga-2 pa-2">
+            <div class="d-flex ga-2">
+                <div class="d-inline-flex">
+                    <v-img
+                        :width="240"
+                        aspect-ratio="4/3"
+                        src="https://cdn.vuetifyjs.com/images/parallax/material.jpg"
+                    ></v-img>
+                </div>
+                <div class="d-flex flex-column">
+                    <router-link :to="link">
+                        <span class="text-body-1">{{ name }}</span>
+                    </router-link>
+                    <ProductPrice :price="price" :retail-price="retailPrice" />
+                </div>
+            </div>
+            <div v-if="isShoppingCartItem" class="d-flex ga-2">
```

Comment:
tbh this feels a bit like this component serves two purposes: just displaying it and also this shopping cart thing
generally speaking, this violates separation of concerns a bit
I personally would probably have added a slot to this component, which by default is empty, but where for the shopping cart use case, the new stuff can be added, but feel free to ignore this if you deem it too much work

Reply:
How about I open a Gropius Issue for this so that I can change things later? Because I think there a more pressing issues right now.

Reply:
fine for me